themes/theme-gmd/components/ProductCard/index.jsx

```diff
@@ -79,3 +82,4 @@ ProductCard.propTypes = {
 };
 
 export default ProductCard;
+/* eslint-enable extra-rules/no-single-line-objects */
```

Comment:
Please don't deactivate these rules constantly. We decided on these rules in the COP frontend once and everyone that writes frontend code should follow them. No single line objects. This is for readability of the code.